packages/jupytercad-extension/src/mainview.tsx

```diff
@@ -236,9 +236,9 @@ export class MainView extends React.Component<IProps, IStates> {
 
       this._scene = new THREE.Scene();
 
-      this._scene.add(new THREE.AmbientLight(0xffffff, 0.8)); // soft white light
+      this._scene.add(new THREE.AmbientLight(0xffffff, 0.5)); // soft white light
 
-      const light = new THREE.SpotLight(0xffffff, 0.2);
+      const light = new THREE.PointLight(0xffffff, 1);
       light.castShadow = true;
```

Comment:
I think we should disable the shadowmap computing. We're not using it so we could save some perf here.